src/ErrorHandler.php

```diff
@@ -40,6 +38,8 @@ final class ErrorHandler
     private string $memoryReserve = '';
     private bool $debug = false;
     private ?string $workingDirectory = null;
+    private bool $enabled = false;
```

Comment:
Is there a need for two flags?

Reply:
Yes. 

- `enabled` — handle or not errors/exceptions in handlers.
- `isHandlersInitialized` — for prevent double registration of handlers.